docs/source/quickstart.mdx

```diff
@@ -0,0 +1,389 @@
+---
+title: "Federation quickstart (preview)"
+sidebar_title: "Part 1 - Local schema composition"
+description: "Part 1 - Local schema composition"
+---
+
+import {Button} from '@apollo/space-kit/Button';
+import {Link} from 'gatsby';
+import {ExpansionPanel} from 'gatsby-theme-apollo-docs';
+
+> Instead of or in addition to completing this tutorial, you can [clone our example gateway on GitHub](https://github.com/apollographql/supergraph-demo). The GitHub example is **not** identical to the gateway you create in this tutorial, but it demonstrates many of the same concepts.
+
+Hello! This tutorial gets you up and running with Apollo Federation. Unlike other articles in this documentation, this article uses some Apollo tools and terminology that are still in **preview**, and therefore this content is not yet recommended for use in a production environment.
+
+We expect these tools to leave preview soon, so this tutorial is a great way to begin learning about federation using upcoming best practices.
+
+> **This tutorial requires Node.js version 12 or later.** 
+
+## Federation concepts
+
+In a federated architecture, multiple GraphQL APIs are **composed** into a single federated data graph. The individual APIs are called **subgraphs**, and they're composed into a **supergraph**:
+
+```mermaid
+graph BT;
+  gateway(["Supergraph (A + B + C)"]);
+  serviceA[Subgraph A];
+  serviceB[Subgraph B];
+  serviceC[Subgraph C];
+  gateway --- serviceA & serviceB & serviceC;
+```
+
+Usually, each subgraph corresponds to a different service in your backend. The supergraph is then represented by a **gateway**, which routes each incoming query to the appropriate combination of subgraphs and returns the combined result.
+
+The supergraph's schema is the combination of each subgraph's schema, plus some special federation-specific directives. This architecture enables clients to query data from multiple subgraphs simultaneously, just by querying the supergraph.
```

Comment:
I feel like we're selling Federation short here, particularly if I'm new to the concept. Maybe we could link to the Federation spec and mention some of the benefits of Federation vs simply combining schemas into a union?